float64 angle_min
float64 angle_max
int32 angle_min_pwm
int32 angle_max_pwm
int32 channel
int32 board
